docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -0,0 +1,116 @@
+.. _arch_overview_aggregate_cluster:
+
+Aggregate Cluster
+=================
+
+Aggregate cluster is used for failover between clusters with different configuration, e.g., from EDS upstream cluster to STRICT_DNS upstream cluster, 
```

Comment:
nit: please reflow to 100 columns here and elsewhere, it's easier to read in GH and the column limit we use elsewhere.

Reply:
Thanks! Good to know that! I was wondering how should I format the rst doc.